db/migrate/20210112125011_create_event_attendees.rb

```diff
@@ -0,0 +1,9 @@
+class CreateEventAttendees < ActiveRecord::Migration[6.1]
+  def change
+    create_table :event_attendees do |t|
+      t.belongs_to :attendee
+      t.belongs_to :attended_event
+      t.timestamps
+    end
+  end
```

Comment:
Add an index to your attendee and attended_event, since it allows data to be quickly located without having to search every row in a database table every time a database table is accessed
```
      t.belongs_to :attendee, index: true
      t.belongs_to :attended_event, index: true
```

Reply:
Added Index by creating a new migration "AddIndexToEventsAttendee"